packages/file_selector/file_selector_platform_interface/pubspec.yaml

```diff
@@ -3,23 +3,23 @@ description: A common platform interface for the file_selector plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/file_selector/file_selector_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.3+1
+version: 2.0.0-nullsafety.0
 
 dependencies:
   flutter:
     sdk: flutter
   meta: ^1.0.5
-  http: ^0.12.0+1
-  plugin_platform_interface: ^1.0.1
-  cross_file: ^0.2.0
+  http: ^0.13.0-nullsafety.0
+  plugin_platform_interface: 1.1.0-nullsafety.2
```

Comment:
This needs a `^` (see the error message on the failing `publishable` CI bot)

Reply:
Yes will do that. Thanks, can expect commit in 15mins